bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java

```diff
@@ -325,29 +323,25 @@ private void cacheNewState(StateUpdate newState) {
     private @Nullable Integer getCurrentColorTemp(@Nullable State groupState) {
         Integer colorTemp = lastSentColorTemp;
         if (colorTemp == null && groupState != null) {
-            colorTemp = groupState.getColorTemperature();
+            return groupState.getColorTemperature();
```

Comment:
Is it really a better practice to multiply return in a method ?

Reply:
I think the way how to do it at best can be discussed controversial. And we should not focus on that but on the functional correctness.

I am using early returns because from my pov it is easier to maintain and improves readability. You clearly can see exit points in a method or function. It might increased performance as well. But as I said we should not focus on it. If you like I will revert it.

Reply:
As I said, this was a minor remark and my only remark.